ops/ccs-ops-misc/load_test/common/stats/stats_config.py

```diff
@@ -0,0 +1,135 @@
+import dataclasses
+from enum import Enum
+import re
+from dataclasses import dataclass
+from typing import Optional, Type, TypeVar
+
+E = TypeVar('E', bound=Enum)
```

Comment:
This is actually how Python handles [generic type hinting](https://docs.python.org/3/library/typing.html#typing.TypeVar). This particular generic type, `E`, is used in the method signature for `__enum_from_val()`:

```python
def __enum_from_val(val: str, enum_type: Type[E], field_name: str) -> E:
```

Basically this makes the signature for the method more accurate so that linters and other tooling are able to discern that the method's return type is "any type extending from Enum".

Reply:
Neat! I'm still new to python's type hinting, so good to learn a bit more.